warehouse/templates/pages/help.html

```diff
@@ -20,11 +20,11 @@
     <div class="narrow-container">
       <h2>Common Questions</h2>
 
-      <h3 id="installing" class="common-question">What is a package and how do I install one?</h3>
+      <h3 id="installing" class="common-question">What is a project and how do I install one?</h3>
 
-      <p>A 'package' in the context of PyPI is a bundle of software to be installed.</p>
-      <p>Packages on PyPI are developed and shared by other members of the Python community so that you can use them in your own projects.</p>
-      <p>To learn how to install packages, visit the <a href="//packaging.python.org/en/latest/installing/">installation tutorial</a> on the <a href="//packaging.python.org/">Python Packaging User Guide</a>.</p>
+      <p>A 'project' in the context of PyPI is a bundle of software to be installed.</p>
+      <p>Projects on PyPI are developed and shared by other members of the Python community so that you can use them in your own projects.</p>
+      <p>To learn how to install projects, visit the <a href="//packaging.python.org/en/latest/installing/">installation tutorial</a> on the <a href="//packaging.python.org/">Python Packaging User Guide</a>.</p>
```

Comment:
I think this section works better as "package", again because it's the end user focused "thing I want to install".